### ISSUE TITLE
AUC_ROC_Score had not changed after the new release

### ISSUE THREAD

**Issue by Juliyan96** [BioSystemsUM/DeepMol#47]
Hi Joao,
I tried calculating the metrics after the AUC_ROC update. But it seems like the ROC values had not changed when calculating for the test set. I have attached the code below. I used both `pip install deepmol` and `pip install deepmol[all]` to install the package. Is it a problem with my code?

Code:
```
from boruta.boruta_py import BorutaPy
from deepmol.feature_selection.base_feature_selector import BorutaAlgorithm
from deepmol.compound_featurization import MorganFingerprint
from deepmol.compound_featurization import MACCSkeysFingerprint
from deepmol.loaders.loaders import CSVLoader
from deepmol.feature_selection import LowVarianceFS
from deepmol.splitters.splitters import SingletaskStratifiedSplitter
from deepmol.models.sklearn_models import SklearnModel
from deepmol.metrics.metrics import Metric
from deepmol.metrics.metrics_functions import precision_score, accuracy_score, confusion_matrix, classification_report, roc_auc_score
from deepmol.imbalanced_learn.imbalanced_learn import SMOTEENN
import numpy as np
from deepmol.standardizer.custom_standardizer import CustomStandardizer, heavy_standardisation
standardizer2 = CustomStandardizer(heavy_standardisation).standardize(dataset)
dataset = CSVLoader(dataset_path='/content/drive/MyDrive/2. Dataset_Deepmol.csv', mols_field='Smiles', labels_fields='Class', id_field='ID')
dataset = dataset.create_dataset()
dataset.get_shape()
dataset = MorganFingerprint().featurize(dataset)
print(dataset.X)
print(dataset.y)
print(dataset.mols)
print(dataset.get_shape())
dataset = LowVarianceFS(0.15).select_features(dataset)
dataset.get_shape()
splitter = SingletaskStratifiedSplitter()
train_dataset, valid_dataset, test_dataset = splitter.train_valid_test_split(dataset=dataset, frac_train=0.8,
                                                                             frac_valid=0.2, frac_test=0.0)
print('Train:')
train_dataset.get_shape()
print('\nValidation:')
valid_dataset.get_shape()
from sklearn.ensemble import RandomForestClassifier
rf = RandomForestClassifier()
model = SklearnModel(model=rf)
model.fit(train_dataset)
model.cross_validate(dataset, Metric(roc_auc_score), folds=10)
metrics = [Metric(roc_auc_score), Metric(precision_score), Metric(accuracy_score), Metric(confusion_matrix), 
           Metric(classification_report)]
print('Training Dataset: ')
train_score = model.evaluate(train_dataset, metrics)
print('Validation Dataset: ')
valid_score = model.evaluate(valid_dataset, metrics)
```
Training Dataset: 
roc_auc_score: 
 0.8672906729170093
precision_score: 
 0.9095890410958904
accuracy_score: 
 0.9598326032540676
confusion_matrix: 
 [[22217   231]
 [  796  2324]]
classification_report: 
               precision    recall  f1-score   support

           0       0.97      0.99      0.98     22448
           1       0.91      0.74      0.82      3120

    accuracy                           0.96     25568
   macro avg       0.94      0.87      0.90     25568
weighted avg       0.96      0.96      0.96     25568

WARNING: task averager  cannot perform reduce with flexible type
Validation Dataset: 
roc_auc_score: 
 0.5669997006307794
precision_score: 
 0.3682795698924731
accuracy_score: 
 0.8627972465581978
confusion_matrix: 
 [[5378  235]
 [ 642  137]]
classification_report: 
               precision    recall  f1-score   support

           0       0.89      0.96      0.92      5613
           1       0.37      0.18      0.24       779

    accuracy                           0.86      6392
   macro avg       0.63      0.57      0.58      6392
weighted avg       0.83      0.86      0.84      6392

It seems like the AUC_ROC values are off compared to the weighted averages. The results are also similar to that before it was updated.
Thanks.

**Comment by jcorreia11** [comment]
Hi Juliyan, can you make sure you have the new version installed (0.0.4.3b0)?
It is possible that a cached version is being installed instead of the new version.
If so, install the new version (`pip install deepmol==0.0.4.3b0` ) and let me know if the problem persists.

**Comment by Juliyan96** [comment]
Hi Joao,
I installed the latest version and it seemed to have worked. Thank you for your prompt response.
Regards,
Juliyan
